packages/bridge-ui-v2/config/schemas/configuredBridges.schema.json

```diff
@@ -31,20 +31,39 @@
               "erc1155VaultAddress": {
                 "type": "string"
               },
-              "crossChainSyncAddress": {
+              "taikoAddress": {
                 "type": "string"
               },
               "signalServiceAddress": {
                 "type": "string"
+              },
+              "hops": {
+                "type": "array",
+                "default": [],
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "chainId": {
+                      "type": "number"
+                    },
+                    "taikoAddress": {
+                      "type": "string"
+                    },
+                    "signalServiceAddress": {
+                      "type": "string"
+                    }
+                  }
+                }
               }
             },
             "required": [
               "bridgeAddress",
               "erc20VaultAddress",
               "erc721VaultAddress",
               "erc1155VaultAddress",
-              "crossChainSyncAddress",
-              "signalServiceAddress"
+              "taikoAddress",
+              "signalServiceAddress",
+              "hops"
```

Comment:
Hops should not be required. L1-L2 does not need hops afaik.

Reply:
Updated.